app-frontend/src/app/components/tools/toolCreateModal/toolCreateModal.controller.js

```diff
@@ -46,7 +46,7 @@ export default class ToolCreateModalController {
         let expressionTree = null;
         try {
             expressionTree = mathjs.parse(this.definitionExpression);
-            this.toolBuffer.definition = this.transformNode(expressionTree);
```

Comment:
Looks like this function still exists but is no longer used -- should it be removed?